theme.json

```
@@ -97,6 +97,63 @@
 			],
 			"units": ["%", "px", "em", "rem", "vh", "vw"]
 		},
+		"typography": {
+			"defaultFontSizes": false,
+			"fluid": true,
+			"fontSizes": [
+				{
+					"fluid": false,
+					"name": "Extra Small",
+					"size": "14px",
+					"slug": "x-small"
+				},
+				{
+					"fluid": {
+						"max": "18px",
+						"min": "18px"
+					},
+					"name": "Small",
+					"size": "1rem",
+					"slug": "small"
+				},
+				{
+					"fluid": {
+						"max": "22px",
+						"min": "18px"
```

If we go with `rem`, this should be:

```suggestion
						"max": "1.375rem",
						"min": "1.125rem"
```